### PR TITLE
fixed docstring in eigsh_lanczos

### PR DESCRIPTION
**Pull request by mganahl**

### CODE REVIEW

File: tensornetwork/backends/base_backend.py [hunk 1]
```diff
@@ -310,9 +310,15 @@ def eigsh_lanczos(self,
...
       delta: Stopping criterion for Lanczos iteration.
+<<<<<<< HEAD
         If a Krylov vector `x_n` has an L2 norm ||x_n|| < delta, the iteration 
         is stopped. It means that an (approximate) invariant subspace has 
         been found.
+=======
+        If two successive Krylov vectors `x_m` and `x_n`
+        have an overlap abs(<x_m|x_n>) < delta, the iteration is stopped.
+        It means that an (approximate) invariant subspace has been found.
+>>>>>>> parent of d2afc92... fixed docstrings of eigsh_lanczos
```
Comment (Thenerdstation) [inline tensornetwork/backends/base_backend.py]: Fix this merge issue.

File: tensornetwork/backends/pytorch/pytorch_backend.py [hunk 2]
```diff
@@ -167,9 +167,15 @@ def eigsh_lanczos(self,
...
       delta: Stopping criterion for Lanczos iteration.
+<<<<<<< HEAD
         If a Krylov vector `x_n` has an L2 norm ||x_n|| < delta, the iteration 
         is stopped. It means that an (approximate) invariant subspace has been 
         found.
+=======
+        If two successive Krylov vectors `x_m` and `x_n`
+        have an overlap abs(<x_m|x_n>) < delta, the iteration is stopped.
+        It means that an (approximate) invariant subspace has been found.
+>>>>>>> parent of d2afc92... fixed docstrings of eigsh_lanczos
```
Comment (Thenerdstation) [inline tensornetwork/backends/pytorch/pytorch_backend.py]: Fix this too.

**Comment by mganahl** [comment]
oops sorry, had to revert the commits because I confused two branches ...

File: tensornetwork/backends/base_backend.py [hunk 3]
```diff
@@ -301,9 +301,9 @@ def eigsh_lanczos(self,
         as stopping criterion between two diagonalization steps of the
         tridiagonal operator.
       delta: Stopping criterion for Lanczos iteration.
-        If two successive Krylov vectors `x_m` and `x_n`
-        have an overlap abs(<x_m|x_n>) < delta, the iteration is stopped.
-        It means that an (approximate) invariant subspace has been found.
+        If a Krylov vector `x_n` has an L2 norm ||x_n|| < delta, the iteration
```
Comment (Thenerdstation) [inline tensornetwork/backends/base_backend.py]: :math this like we did in `split_node`

File: tensornetwork/backends/numpy/numpy_backend.py [hunk 4]
```diff
@@ -160,9 +160,9 @@ def eigsh_lanczos(self,
         as stopping criterion between two diagonalization steps of the
         tridiagonal operator.
       delta: Stopping criterion for Lanczos iteration.
-        If two successive Krylov vectors `x_m` and `x_n`
-        have an overlap abs(<x_m|x_n>) < delta, the iteration is stopped.
-        It means that an (approximate) invariant subspace has been found.
+        If a Krylov vector `x_n` has an L2 norm ||x_n|| < delta, the iteration
```
Comment (Thenerdstation) [inline tensornetwork/backends/numpy/numpy_backend.py]: :math

File: tensornetwork/backends/pytorch/pytorch_backend.py [hunk 5]
```diff
@@ -156,9 +156,9 @@ def eigsh_lanczos(self,
         as stopping criterion between two diagonalization steps of the
         tridiagonal operator.
       delta: Stopping criterion for Lanczos iteration.
-        If two successive Krylov vectors `x_m` and `x_n`
-        have an overlap abs(<x_m|x_n>) < delta, the iteration is stopped.
-        It means that an (approximate) invariant subspace has been found.
+        If a Krylov vector `x_n` has an L2 norm ||x_n|| < delta, the iteration
```
Comment (Thenerdstation) [inline tensornetwork/backends/pytorch/pytorch_backend.py]: :math

**Pull request merged**